### PR TITLE
Add support for RemoteIPInternalProxy

### PR DESCRIPTION
**Pull request by mariusvw**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Add support for RemoteIPInternalProxy in mod_remoteip config settings

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -444,6 +444,9 @@ apache:
     RemoteIPTrustedProxy:
       - 10.0.8.0/24
       - 127.0.0.1
+    RemoteIPInternalProxy
```
Comment (noelmcloughlin) [inline pillar.example:447]: Needs a colon.
```
 [CRITICAL] Rendering SLS 'apache' failed, render error:
       could not find expected ':'
```

File: test/salt/pillar/modules.sls [hunk 2]
```diff
@@ -427,6 +427,9 @@ apache:
     RemoteIPTrustedProxy:
       - 10.0.8.0/24
       - 127.0.0.1
+    RemoteIPInternalProxy
```
Comment (noelmcloughlin) [inline test/salt/pillar/modules.sls:430]: This too.

**Review by noelmcloughlin — CHANGES_REQUESTED** [review]
LGTM, and useful.  Just a small issue to fix. 

Please also change commit message from "Add support for RemoteIPInternalProxy" to "feat(trusted_proxy): Add support for RemoteIPInternalProxy" for compatibility with semantic release.

File: test/salt/pillar/modules.sls (hunk 2 above)
Comment (mariusvw) [reply test/salt/pillar/modules.sls]: Updated

Comment (noelmcloughlin) [reply test/salt/pillar/modules.sls]: Thanks. Please update commit messages so https://github.com/semantic-release/semantic-release works.  I suggested "**feat(trusted_proxy):** Add support for RemoteIPInternalProxy" and 2nd could be "**fix(syntax):** add semicolon".

**Review by noelmcloughlin — CHANGES_REQUESTED** [review]
Thanks. Please update commit messages so https://github.com/semantic-release/semantic-release works. I suggested "feat(trusted_proxy): Add support for RemoteIPInternalProxy" and 2nd could be "fix(syntax): add semicolon".

**Comment by mariusvw** [comment]
Continues here https://github.com/saltstack-formulas/apache-formula/pull/300

**Pull request closed without merging**